src/triggers/budget.ts

```diff
@@ -0,0 +1,17 @@
+export namespace Budget {
+    type Message = {
+        billing_account_id: string;
+        budget_id: string;
+        budget_display_name: string;
+        amount: number;
+        currency: "RUB" | string; // TODO should be enum,
```

Comment:
`"RUB"`? Why not just `string`? As well as for `period` and `threshold_type`?

Reply:
To show that what kind of enum it might be for future developing

Reply:
I'm not sure that Http.Result is required for triggers, it does not break anything if something else is returned